zeppelin-web/src/app/notebook/notebook.html

```diff
@@ -13,6 +13,7 @@
 -->
 <!-- Here the controller <NotebookCtrl> is not needed because explicitly set in the app.js (route) -->
 <div id="actionbar" ng-include src="'app/notebook/notebook-actionBar.html'"></div>
+<div id="note-revisions-comparator-modal-container" ng-include src="'app/notebook/revisions-comparator.html'"></div>
```

Comment:
Hi, can we use angular1's component instead of using ng-include? 

Here are few reasons. 

- ng-include can't be handled in webpack. It means we can't take advantage of fast re-build and HMR
- it's blocker when we are moving to other stacks. (whatever it's is)

Here is ES6 based component guide and an example in Zeppelin

- https://github.com/toddmotto/angularjs-styleguide
- https://github.com/apache/zeppelin/tree/master/zeppelin-web/src/app/jobmanager/job